www/src/pages/blog/introducing-astro.astro

```diff
@@ -13,9 +13,10 @@ let publishDate = 'Tuesday, June 8 2021';
 let author = 'fred';
 let heroImage = '/social.jpg';
 let permalink = 'https://astro.build/blog/introducing-astro';
+let lang = 'en';
 ---
 
-<html>
+<html lang={ lang || 'en' }>
```

Comment:
Any reason for `||` here versus `??` elsewhere?

Reply:
Whoops, thanks for catching that